src/OpenTelemetry.Proto/OpenTelemetry.Proto.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <Description>OpenTelemetry protos for OpenTelemetry .NET</Description>
+    <MinVerTagPrefix>proto-</MinVerTagPrefix>
+
+    <ExcludeFromPublicApiAnalyzer>true</ExcludeFromPublicApiAnalyzer>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OpenTelemetryProtoVersion>0.3.0</OpenTelemetryProtoVersion>
```

Comment:
A suggestion (https://github.com/open-telemetry/opentelemetry-dotnet/pull/1949#issuecomment-810739808) on a prior iteration of this idea is to have a CI job automate the generation of new versions of the OpenTelemetry.Proto package. My thought is we might be able to introduce this in the future. For now, bumping the version here is how it's done.